src/plugins/exception-helpers.ts

```diff
@@ -42,7 +42,7 @@ async function htmlNotFound (ctx: MojoContext): Promise<boolean> {
 }
 
 async function httpException (ctx: MojoContext, error: Error): Promise<boolean> {
-  ctx.log.error(error.stack);
+  ctx.log.error(error.stack ?? 'error.stack is undefined for ' + error.message);
```

Comment:
Why does the user care if the stack is missing?

Reply:
Also would need a test case.

Reply:
changed to `as string`